### ISSUE TITLE
SCSSLint::Linter::Comment - Length must be more than 0, passed -38

### ISSUE THREAD

**Issue by gpoul** [brigade/scss-lint#887]
linting this:

```
@mixin make-grid($class) {
  @if ($class == "lg")
    { $grid-columns-lpo: $grid-columns + 3;  } /*sev */
}
```

results in this:

```
SCSSLint::Linter::Comment raised unexpected error linting file repro.scss: 'Length must be more than 0, passed -38'
/home/gpoul/.gem/ruby/2.1.0/gems/scss_lint-0.52.0/lib/scss_lint/location.rb:14:in `initialize'
/home/gpoul/.gem/ruby/2.1.0/gems/scss_lint-0.52.0/lib/scss_lint/linter.rb:78:in `new'
/home/gpoul/.gem/ruby/2.1.0/gems/scss_lint-0.52.0/lib/scss_lint/linter.rb:78:in `location_from_range'
/home/gpoul/.gem/ruby/2.1.0/gems/scss_lint-0.52.0/lib/scss_lint/linter.rb:173:in `extract_location'
/home/gpoul/.gem/ruby/2.1.0/gems/scss_lint-0.52.0/lib/scss_lint/linter.rb:61:in `add_lint'
/home/gpoul/.gem/ruby/2.1.0/gems/scss_lint-0.52.0/lib/scss_lint/linter/comment.rb:7:in `visit_comment'
/home/gpoul/.gem/ruby/2.1.0/gems/sass-3.4.23/lib/sass/tree/visitors/base.rb:36:in `visit'
/home/gpoul/.gem/ruby/2.1.0/gems/scss_lint-0.52.0/lib/scss_lint/linter.rb:139:in `visit'
/home/gpoul/.gem/ruby/2.1.0/gems/scss_lint-0.52.0/lib/scss_lint/linter.rb:150:in `block in visit_children'
/home/gpoul/.gem/ruby/2.1.0/gems/scss_lint-0.52.0/lib/scss_lint/linter.rb:148:in `each'
/home/gpoul/.gem/ruby/2.1.0/gems/scss_lint-0.52.0/lib/scss_lint/linter.rb:148:in `visit_children'
/home/gpoul/.gem/ruby/2.1.0/gems/sass-3.4.23/lib/sass/tree/visitors/base.rb:38:in `visit'
/home/gpoul/.gem/ruby/2.1.0/gems/scss_lint-0.52.0/lib/scss_lint/linter.rb:139:in `visit'
/home/gpoul/.gem/ruby/2.1.0/gems/scss_lint-0.52.0/lib/scss_lint/linter.rb:150:in `block in visit_children'
/home/gpoul/.gem/ruby/2.1.0/gems/scss_lint-0.52.0/lib/scss_lint/linter.rb:148:in `each'
/home/gpoul/.gem/ruby/2.1.0/gems/scss_lint-0.52.0/lib/scss_lint/linter.rb:148:in `visit_children'
/home/gpoul/.gem/ruby/2.1.0/gems/sass-3.4.23/lib/sass/tree/visitors/base.rb:38:in `visit'
/home/gpoul/.gem/ruby/2.1.0/gems/scss_lint-0.52.0/lib/scss_lint/linter.rb:139:in `visit'
/home/gpoul/.gem/ruby/2.1.0/gems/scss_lint-0.52.0/lib/scss_lint/linter.rb:41:in `run'
/home/gpoul/.gem/ruby/2.1.0/gems/scss_lint-0.52.0/lib/scss_lint/runner.rb:53:in `run_linter'
/home/gpoul/.gem/ruby/2.1.0/gems/scss_lint-0.52.0/lib/scss_lint/runner.rb:35:in `block in find_lints'
/home/gpoul/.gem/ruby/2.1.0/gems/scss_lint-0.52.0/lib/scss_lint/runner.rb:33:in `each'
/home/gpoul/.gem/ruby/2.1.0/gems/scss_lint-0.52.0/lib/scss_lint/runner.rb:33:in `find_lints'
/home/gpoul/.gem/ruby/2.1.0/gems/scss_lint-0.52.0/lib/scss_lint/runner.rb:19:in `block in run'
/home/gpoul/.gem/ruby/2.1.0/gems/scss_lint-0.52.0/lib/scss_lint/runner.rb:18:in `each'
/home/gpoul/.gem/ruby/2.1.0/gems/scss_lint-0.52.0/lib/scss_lint/runner.rb:18:in `run'
/home/gpoul/.gem/ruby/2.1.0/gems/scss_lint-0.52.0/lib/scss_lint/cli.rb:67:in `scan_for_lints'
/home/gpoul/.gem/ruby/2.1.0/gems/scss_lint-0.52.0/lib/scss_lint/cli.rb:59:in `act_on_options'
/home/gpoul/.gem/ruby/2.1.0/gems/scss_lint-0.52.0/lib/scss_lint/cli.rb:33:in `run'
/home/gpoul/.gem/ruby/2.1.0/gems/scss_lint-0.52.0/bin/scss-lint:7:in `<top (required)>'
/home/gpoul/.gem/ruby/2.1.0/bin/scss-lint:23:in `load'
/home/gpoul/.gem/ruby/2.1.0/bin/scss-lint:23:in `<main>'
Report this bug at https://github.com/brigade/scss-lint/issues
 
To help fix this issue, please include:
- The above stack trace
- SCSS-Lint version: 0.52.0
- Sass version: 3.4.23
- Ruby version: 2.1.5
```

**Comment by trotzig** [comment]
Ouch. Can you help narrow this down by perhaps reformatting that if-statement a little and see if there is any change?

**Comment by gpoul** [comment]
@trotzig what I found was that this goes away as soon as you add a new line after the /* but simply inserting a blank doesn't fix it.

**Comment by trotzig** [comment]
Okay, good to know. 

I don't have much time to focus on this now, so if you want to take a stab at it, I'd be happy for a PR.

**Comment by srawlins** [comment]
This is caused by https://github.com/sass/sass/issues/2284. I wouldn't wait on a fix, but we can work around it in scss-lint.
